ddp-workspace/projects/ddp-testboston/src/app/components/header/header.component.html

```diff
@@ -24,12 +24,6 @@
                         Navigation.WhoWeAre
                     </a>
                 </li>
-                <li class="navigation__item">
-                    <a class="header-link Link_no-wrap" routerLink="/" fragment="stay-informed"
-                        (click)="scrollToAnchor('stay-informed')" translate>
-                        Navigation.StayInformed
-                    </a>
-                </li>
```

Comment:
You've forgotten to remove one more mailing list: 79-84 lines.
And you have to remove the mailing list from the footer.

Reply:
Oooh whoops!  Thank you.